example/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/bonitoo-io/influxdb3-go/influx"
+)
+
+func main() {
+	// Use env variables to initialize client
+	url := os.Getenv("INFLUXDB_URL")
+	token := os.Getenv("INFLUXDB_TOKEN")
+	bucket := os.Getenv("INFLUXDB_BUCKET")

```

Comment:
Please use DATABASE instead of bucket:
```suggestion
	database := os.Getenv("INFLUXDB_ DATABASE")

```